slave.mk

```diff
@@ -668,6 +714,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 		rm -f $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).sh
 		rm -f $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).service
 	)
+	rm -f updategraph.service
```

Comment:
Why?

Reply:
`updategraph.service` wasn't getting deleted after the build finished